secure_message/common/utilities.py

```diff
@@ -78,30 +79,53 @@ def process_paginated_list(paginated_list, host_url, user, message_args, endpoin
 
 
 def add_to_details(messages):
-    """Adds a @msg_to key every message in a list of messages.
+    """Adds a @msg_to key to every message in a list of messages.
     Every msg_to uuid is resolved to include details of the user.
 
     If the call for the internal user id fails, an exception will be thrown.
     If the external user id cannot be found in the list that we got from the party service.  There
     won't be a @msg_to value returned in the payload.  The API documentation notes that these elements
     aren't guaranteed to be provided so we're not breaking the contract by doing this.
+
+    Note: Several of these lines of code could be combined into a more succinct view, spreading them out
+    is deliberate so that log stack traces are better able to identify the cause of log errors
     """
+
     external_user_details = {}
-    for user in party.get_users_details(get_external_user_uuid_list(messages)):
+
+    external_users = get_external_user_uuid_list(messages)
+
+    for user in party.get_users_details(external_users):
```

Comment:
Why split this over 2 lines if we're not going to add any extra logging?

Reply:
Only so that any stack trace specifies the exact culprit. No functional difference.